pkg/db-repo/model.go

```diff
@@ -35,8 +35,8 @@ type TimestampAware interface {
 }
 
 type Timestamps struct {
-	UpdatedAt *time.Time
-	CreatedAt *time.Time
+	UpdatedAt *time.Time `gorm:"type:datetime"`
```

Comment:
are those gorm hint required now? or is it just a nicety to have them?

Reply:
They help for validation / auto-generation of the correct schemas